blockchain/storage/memory/batch.go

```diff
@@ -0,0 +1,35 @@
+package memory
+
+import (
+	"github.com/0xPolygon/polygon-edge/helper/hex"
+)
+
+type batchMemory struct {
+	db           map[string][]byte
+	keysToDelete [][]byte
+	valuesToPut  [][2][]byte
```

Comment:
Is 2 here denoting key+value byte slices? So the first slice represents `key` and the second one represents `value`?

Reply:
yes, exactly. `batchMemory` is used mostly for the tests. maybe we should add comment like:
`// slice representing (key, value) pairs that will be added/updated on Write call